plugins/react-native/react-native.plugin.zsh

```diff
@@ -31,7 +31,6 @@ alias rnios12='react-native run-ios --simulator "iPhone 12"'
 alias rnios12p='react-native run-ios --simulator "iPhone 12 Pro"'
 alias rnios12pm='react-native run-ios --simulator "iPhone 12 Pro Max"'
 
-
```

Comment:
Why is this being removed?

Reply:
@esthor 

Fix: Multiple consecutive blank lines, following Markdown documentation.
Ref: [MD012](https://github.com/DavidAnson/markdownlint/blob/v0.25.1/doc/Rules.md#md012)

Reply:
Ah, awesome. I see that now the the extra line has been there for years. 😅  Nice catch! 

(I'm still confused why the diff view on github shows a `-` being removed instead of just a blank line 🧐 , but that's a different hill to climb 🤷 😂 )